common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -841,4 +841,4 @@ describe 'Problem', ->
       jasmine.clock().tick(64000)
       expect(@problem.poll.calls.count()).toEqual(6)
 
-      expect($('.capa_alert').text()).toEqual("The grading process is still running. Refresh the page to see updates.")
+      expect($('.notification-gentle-alert .notification-message').text()).toEqual("The grading process is still running. Refresh the page to see updates.")
```

Comment:
Given that this text is always present, this test is doing much. Can we check that the class has been properly updated so that the HTML will be visible?